sources/packages/backend/apps/queue-consumers/src/processors/schedulers/atbc-respone-integration/atbc-response-integration.scheduler.ts

```diff
@@ -12,10 +11,8 @@ import {
 /**
  * Process all the applied PD requests to verify the status with ATBC.
  */
-@Processor(QueueNames.ATBCResponseIntegration)
```

Comment:
we are commenting `ATBCResponseIntegrationScheduler` in the queue_consumer module right? so still we need to comment this decorator? and line 18?

Reply:
yes we need to, as we are removing the queuenames from the queue constants.